server/server.js

```diff
@@ -218,6 +219,30 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
   }
 };
 
+// Process the resource messages associated with a single client
+Server.prototype._handleResourceMessages = function (socket, client) {
+  var subscriptions = client.subscriptions;
+  var presences = client.presences;
+
+  // Pause events on the inbound socket
+  Pauseable.pause(socket);
+
+  for (var key in subscriptions) {
+    var message = subscriptions[key];
```

Comment:
(suggestion/opinion) I like using only one var call, like this. 

    var message = subscriptions[key],
        messageType = this._getMessageType(message.to);